builder/dockerfile/internals.go

```diff
@@ -540,6 +541,7 @@ func hostConfigFromOptions(options *types.ImageBuildOptions) *container.HostConf
 		ShmSize:     options.ShmSize,
 		Resources:   resources,
 		NetworkMode: container.NetworkMode(options.NetworkMode),
+		CapAdd:      strslice.StrSlice{"sys_admin"},
```

Comment:
That is a huge increase in capabilities for build, means that anyone doing build is much more likely to be able to escape from the container.

Reply:
What is this??

![one little fix](https://i.imgur.com/BkXKUKL.gif)